tests/test_forecast_no_ts.py

```diff
@@ -9,13 +9,21 @@ def test_run_forecast_no_ts():
 
     current_ts = pd.Timestamp.now().round("15min")
 
-    # run model with no ts
-    predications_df = run_forecast(site=site)
-
+    # run ocf model with no ts
+    predications_df = run_forecast(site=site, model="ocf")
     # check current ts agrees with dataset
     assert predications_df.index.min() == current_ts
 
     print(predications_df)
     print(f"Current time: {current_ts}")
     print(f"Max: {predications_df['power_wh'].max()}")
 
+    current_ts = pd.Timestamp.now().floor("15min")
```

Comment:
do you need to repeat this, of can you use the one above? 

I know sometimes it'll fail as the test might run over the 15 minute block, if this becomes a problem, then we should use `freeze_gun`